sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -91,6 +91,12 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+BQ_SOURCE_UW_ERROR = (
+    'The Read(BigQuerySource(...)) transform is not supported with newer stack '
+    'features (Fn API, Unified Worker, etc). Please use the transform '
```

Comment:
s/Unified Worker/Dataflow runner V2 I believe.

cc: @robertwb @ananvay for the correct wording here.

Reply:
+1 -- "Dataflow Runner v2"

Reply:
Done. Waiting for their pointers on wording.